depends/packages/packages.mk

```diff
@@ -5,6 +5,7 @@ qt_packages = qrencode protobuf zlib
 
 qt_x86_64_linux_packages:=qt expat dbus libxcb xcb_proto libXau xproto freetype fontconfig libX11 xextproto libXext xtrans
 qt_i686_linux_packages:=$(qt_x86_64_linux_packages)
+qt_arm_linux_packages:=$(qt_x86_64_linux_packages) $(qt_packages)
```

Comment:
What's the role of `$(qt_packages)`, especially given that `qt_i686_linux_packages` doesn't have it?

Reply:
Good findings. I could remove all these changes (I had them in while I was debugging things) and it still seems to run fine. Can you re-run a gitian with b86d49f  ?